include/bout/mesh.hxx

```diff
@@ -597,7 +597,19 @@ class Mesh {
   }
   void addRegion3D(const std::string &region_name, Region<Ind3D> region);
   void addRegion2D(const std::string &region_name, Region<Ind2D> region);
- 
+
+  Ind3D ind2Dto3D(const Ind2D &ind2D, int jz = 0){
+    return Ind3D(ind2D.ind * LocalNz + jz);
+  }
+
+  Ind2D ind3Dto2D(const Ind3D &ind3D){
+    return Ind2D(ind3D.ind / LocalNz);
+  }
+
+  int map3Dto2D(const Ind3D &ind3D){
```

Comment:
Please could you add doxygen comments for these -- specifically, what's the difference between `ind3Dto2D` and `map3Dto2D`?